### PR TITLE
Comments + cleanup in joingroup en addgroup

### PR DESCRIPTION
**Pull request by JochemFB**

### CODE REVIEW

File: Client/ViewModels/JoinGroupPageViewModel.cs [hunk 1]
```diff
@@ -9,51 +9,56 @@
 
 namespace Messenger_Client.ViewModels
 {
-    public class JoinGroupPageViewModel
+    public class JoinGroupPageViewModel : ObservableRecipient
```
Comment (SietzeKoonstra) [inline Client/ViewModels/JoinGroupPageViewModel.cs:12]: Needed?

File: Client/ViewModels/JoinGroupPageViewModel.cs [hunk 2]
```diff
@@ -69,48 +74,74 @@ public JoinGroupPageViewModel()
             JoinGroupButtonCommand = new RelayCommand(SendJoinGroupMessage);
             CancelButtonCommand = new RelayCommand(NavigateToMain);
 
+            //Create empty group list
             GroupList = new ObservableCollection<Group>();
 
+            // Get the groups
             CommunicationHandler.ObtainedRequestedGroups += OnObtainedRequestedGroups;
             CommunicationHandler.SendRequestGroupsMessages();
```
Comment (SietzeKoonstra) [inline Client/ViewModels/JoinGroupPageViewModel.cs:82]: This is maybe a better comment:
Subscribe on event, this event will be raised when the requested groups are obtained. Then send the actual message which request the groups from the server.

**Review by SietzeKoonstra — CHANGES_REQUESTED** [review]

File: Client/ViewModels/JoinGroupPageViewModel.cs (hunk 1 above)
Comment (JochemFB) [reply Client/ViewModels/JoinGroupPageViewModel.cs:12]: yes

Comment (SietzeKoonstra) [reply Client/ViewModels/JoinGroupPageViewModel.cs:12]: ok

**Review by SietzeKoonstra — APPROVED** [review]

**Pull request merged**